### ISSUE TITLE
Is code version  correct?

### ISSUE THREAD

**Issue by TDZ-z** [aprilbian/SEPT#1]
Hello, normal should be False when calculating D1_PSNR in test.py, but it is True in the code.
Is the source code version correct?

**Comment by aprilbian** [comment]
Thanks for the question,

Having 'normal = True' means that the return of the function 'pc_error' will also include point-to-plane distance, i.e., the D2_PSNR. It does not affect the results for D1_PSNR. You can look into the 'utils/pc_error_wrapper.py' for more info.

**Comment by TDZ-z** [comment]
Thanks for your answer! I got it. There are another question:
In the case of SNR=5dB, bottleneck_size=200, your source code test shows D1_PSNR=18.79, D2_PSNR=20.22, which are very different from the result in the paper. 
And I also found that there was an overfitting problem during the training phase, but I didn't modify your core code.
Is there a problem?

**Comment by aprilbian** [comment]
Is it because you modify some hyper parameters? If you run the code with the hyper parameters, e.g., learning rate, batch size unchanged, it should produce the same result... Why not first try reducing the learning rate to see if it works?

**Comment by TDZ-z** [comment]
I didn't modify your hyper parameters, e.g., learning rate=0.001, batch_size=32, I just set your SNR=5 and bottleneck_size=200 according to your paper.
Is there something should be changed when SNR=5 and bottleneck_size=200?
And is your transformer block module consistent with what is stated in the paper?

**Comment by aprilbian** [comment]
No, nothing needs to be changed.
Yes, it is based on the point transformer paper shown in the reference of the paper.

**Comment by aprilbian** [comment]
BTW, please make sure that you decrease the learning rate during the epochs, this should be quite important for a good performance. 

I can provide you with the checkpoints when it gets accepted.

Best,
Chenghong

**Comment by TDZ-z** [comment]
Ok, thanks for your help.😊

Best wishes,
Vi
